test/helper.js

```diff
@@ -173,6 +173,13 @@ function assertEqual (val1, val2, errorStr) {
 
 module.exports.assertEqual = assertEqual;
 
+function assertApproximate (val1, val2, errorStr) {
+    assert(new BN(val2).gte(new BN(val1).mul(new BN(9999)).div(new BN(10000))), errorStr);
```

Comment:
seems to me more accurate if we test

val1.div 10 or 100
to val2.div 10 or 100

it doesn't work?

Reply:
- with small amount this is not correct
- with big amount but small gap like 999999 and 1000000, div and compare values is not true
I updated my code. PTAL